frontend/src/scenes/authentication/TwoFactorSetupModal.tsx

```diff
@@ -0,0 +1,49 @@
+import { useActions, useValues } from 'kea'
+import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
+import { LemonModal } from 'lib/lemon-ui/LemonModal'
+
+import { twoFactorLogic } from './twoFactorLogic'
+import { TwoFactorSetup } from './TwoFactorSetup'
+
+interface TwoFactorSetupModalProps {
+    onSuccess: () => void
+    closable?: boolean
+    required?: boolean
+    forceOpen?: boolean
+}
+
+export function TwoFactorSetupModal({
+    onSuccess,
+    closable = true,
+    required = false,
+    forceOpen = false,
+}: TwoFactorSetupModalProps): JSX.Element {
+    const { isTwoFactorSetupModalOpen } = useValues(twoFactorLogic)
+    const { toggleTwoFactorSetupModal } = useActions(twoFactorLogic)
+
+    return (
+        <LemonModal
+            title="Set up 2FA"
```

Comment:
Not necessary to use an acronym here
```suggestion
            title="Set up two-factor authentication"
```